web/src/app/admin/connectors/[connector]/AddConnectorPage.tsx

```diff
@@ -139,9 +140,9 @@ export default function AddConnector({
 
   const createConnector = async () => {
     const AdvancedConfig: AdvancedConfig = {
-      pruneFreq: pruneFreq || defaultRefresh * 60,
+      pruneFreq: (pruneFreq || defaultPrune) * 60,
       indexingStart,
-      refreshFreq: refreshFreq * 60,
+      refreshFreq: (refreshFreq || defaultRefresh) * 60,
```

Comment:
Nit: Probably fine for now, but worth considering that we may want 0 for a frequency to evaluate to "never refresh / prune" - here 0 would always evaluate to default time (no matter how the backend handles this.)

From my memory, we already handle this on the backend in the API, so a `??` may be more technically accurate.

Reply:
In the interest of speed, I'm putting this one in, Pablo if you have better suggestions feel free to open a new PR. Also, we'll want to support no refresh and no prune options sometime soon (this week)